src/python/pants/build_graph/build_configuration.py

```diff
@@ -32,7 +32,7 @@ class BuildConfiguration:
     _optionables: OrderedSet = field(default_factory=OrderedSet)
     _rules: OrderedSet = field(default_factory=OrderedSet)
     _union_rules: Dict[Type, OrderedSet[Type]] = field(default_factory=dict)
-    _targets: OrderedSet[Type[Target]] = field(default_factory=OrderedSet)
+    _target_types: OrderedSet[Type[Target]] = field(default_factory=OrderedSet)
```

Comment:
Changes to this file should have happened in https://github.com/pantsbuild/pants/pull/9618. I missed them there.